content/en/agent/logs/advanced_log_collection.md

```diff
@@ -31,6 +31,8 @@ Apply log processing rules to a specific log collection configurations to:
 
 To apply a processing rule to all logs collected by a Datadog Agent, see the [Global processing rules](#global-processing-rules) section.
 
+**Note:** As a best practice, Datadog recommends using unified service tagging when applying Docker labels and Kubernetes pod annotations. To learn how to configure your environment with unified tagging, refer to the dedicated [unified service tagging][28] documentation.
```

Comment:
Two notes here:

1. the link is broken as 28 is missing :)
2. I'm not sure this note has much sense in this specific page and let me explain why. This describes advanced capabilities that the agent does (scrubbing, multiline, filtering, ...) and is not in any case link to configured about the collection of the logs or the tagging of the data and not about containerised environment either. Which is why I would likely recommend that we remove this part.